### ISSUE TITLE
How to test your code on VisDA-2017 dataset?

### ISSUE THREAD

**Issue by Externalhappy** [TL-UESTC/UniMoS#3]
Hello, in your paper, you present the results on the VisDA-2017 dataset. However, in the code, there is no implementation for testing on the VisDA-2017.

**Comment by JiaqiZhang-Sengoku** [comment]
Have you resolved this issue? I found that there is no code for other datasets in the code.

**Comment by Externalhappy** [comment]
No, I haven’t resolved it yet. I’ve shifted to another project for now, so I haven’t been able to follow up.
